Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM php:7.0-apache
+
+LABEL maintainer="devops@onix-systems.com"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1 \
+    MYSQL_HOSTNAME=localhost \
+    MYSQL_USER=user \
+    MYSQL_PASSWORD=123456 \
+    MYSQL_DATABASE=rename \
+    CONFIG_FILE=/var/www/html/config/config.php
```

Comment:
You sure that need to use the absolute path name, not a relative?

Reply:
Yes, I am sure, because we always expect config file by this path only